src/main/java/ml/shifu/shifu/core/dtrain/wnd/SerializationUtil.java

```diff
@@ -0,0 +1,141 @@
+package ml.shifu.shifu.core.dtrain.wnd;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SerializationUtil {
+
+    /**
+     * Serialize float array with null and length check.
+     * 
+     * @param out
+     * @param array
+     * @param size
+     * @throws IOException
```

Comment:
param and throws should be commented, otherwise you can remove such comment lines